gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -113,6 +146,10 @@ <h3 class="mdc-layout-grid__cell--span-3">
               {%- else -%}
               <dd>{{ affected.package.name }}</dd>
               {%- endif -%}
+              {%- if 'purl' in affected.package -%}
+              <dt>Purl</dt>
+              <dd>{{ affected.package.purl }}</dd>
+              {%- endif -%}
               {%- endif -%}
```

Comment:
We also a per-affected severity field: https://ossf.github.io/osv-schema/#affectedseverity-field

Reply:
Added a commit for it [c645c99](https://github.com/google/osv.dev/commit/c645c997b19401e31719fefcd6f9956739ac372a)

You might verify it via: `BIT-mediawiki-2020-27621`

sample output:

<img width="1310" alt="Screenshot 2024-05-15 at 9 18 49 AM" src="https://github.com/google/osv.dev/assets/73332835/1fc8946f-31a3-4f3c-9921-e8e666621bb0">

Reply:
The rendering of this is different from the rendering for the top level severity (e.g. on GHSA-76v2-48w6-crxr). Can we make this the same as the top level rendering?

Reply:
update in [212c8f1](https://github.com/google/osv.dev/commit/212c8f12dbfdd63d3ffc6d32ac19a1e032ec80ab)

sample output:
![Screenshot 2024-05-17 at 4 02 35 PM](https://github.com/google/osv.dev/assets/73332835/3f304aad-25f0-426e-8457-6a4f8c57723c)